dt: 0.1
dv: 0.01
dyaw: 0.01
max_speed: 0.42
min_speed: 0.0
max_accel: 5.0
max_yawrate: 1.00
max_dyawrate: 2.0
robot_radius: 0.17

predict_time: 5.0
l_ob_cost_gain: 0.5
speed_cost_gain: 0.7
to_g_goal_cost_gain: 0.3
ignore_to_g_path_cost: 0.8

limit_speed: 0.13
limit_yawrate: 0.70

dwa_only: false
